recipes/dcv_config.rb

```diff
@@ -59,7 +68,7 @@ def allow_gpu_acceleration
   end
 end
 
-if node['platform'] == 'centos' && node['platform_version'].to_i == 7 && node['cfncluster']['cfn_node_type'] == "MasterServer"
+if node['cfncluster']['dcv']['supported_os'].include?("#{node['platform']}#{node['platform_version'].to_i}") && node['cfncluster']['cfn_node_type'] == "MasterServer"
```

Comment:
I don't like this check. Can we be more precise?
Also I don't understand how could it work when `platform_version = 18.04`. Is it included in ['centos7', 'ubuntu18']?

Reply:
`node['platform_version'].to_i` converts `18.04` to `18`, but yes we can be more precise and just use `"#{node['platform']}#{node['platform_version']}"`